Conditional operator/Conditional operator/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace Conditional_operator
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите значение x: ");
+            var x = double.Parse(Console.ReadLine());
+
+            Console.WriteLine("Значение функции равно: " + ConditionalOperator(x));
+
+            Console.ReadKey();
+        }
+
+        public static double ConditionalOperator(double x)
+        {
+            if (Math.Abs(x) > 1)
+            {
+                return 1;
+            }
+            else
+            {
+                if (0 < Math.Abs(x) && Math.Abs(x) <= 1)
+                {
+                    return Math.Abs(x);
+                }
+                else
+                {
+                    return 2;
+                }
+            }
```

Comment:
Этот метод можно было написать компактнее
```suggestion
            if (Math.Abs(x) > 1)
                return 1;
                
            if (x == 0)
                return 2;
                
            return Math.Abs(x);
```